AAT_raspberry/QRScanner.py

```diff
@@ -1,16 +1,12 @@
+import qrtools
```

Comment:
I'm assuming this is the library that you mentioned. Should we keep the changes we made to this library somewhere?